client/src/reviews/ReviewsList.tsx

```diff
@@ -1,36 +1,59 @@
-import React, { ReactElement, useContext } from 'react';
+import React, { ReactNode, useContext, useEffect, useState } from 'react';
 import Contexts from '../contexts/Contexts';
 import ReviewSort from './ReviewSort';
 import ReviewTile from './ReviewTile';
 
 const ReviewsList: React.FC = () => {
-  // const [displayReviews, setDisplayReviews] = useState(false);
-  // const [numDisplayed, setNumDisplayed] = useState(2);
-  // const [filterType, setFilterType] = useState('relevant');
-
-
   const { setModalContent } = useContext(Contexts.ModalContext) || {};
   const { reviews } = useContext(Contexts.ReviewsContext) || {};
-  const child = <div>CHILD EXAMPLE</div>;
 
+  const [displayedReviews, setDisplayReviews] = useState<ReactNode[]>([]);
+  const [numDisplayed, setNumDisplayed] = useState(2);
+
+  const child = <div>CHILD EXAMPLE</div>;
 
-  const onClick = () => {
+  const addReview = () => {
     setModalContent?.call(null, child);
   };
 
+  const moreReviews = () => {
+    if (reviews?.results) {
+      if (numDisplayed === 14 || (reviews.results.length - 1) === numDisplayed) {
+        setNumDisplayed(numDisplayed + 1);
+        setTimeout(() => window.scrollTo(0, document.body.scrollHeight), 0);
+      } else if (numDisplayed < 15 && reviews.results.length - 1 > numDisplayed) {
+        setNumDisplayed(numDisplayed + 2);
+        setTimeout(() => window.scrollTo(0, document.body.scrollHeight), 0);
+      };
+    }
+  };
+
+
+  useEffect(() => {
+    if (reviews?.results) {
+      const displayReviews = [];
+      for (let i = 0; i < numDisplayed; i++) {
```

Comment:
A for of loop might be better here, that way you don't have to write out `i` as much
```suggestion
      for (let result of results.slice(0, numDisplayed)) {
```